include/onnxruntime/core/providers/utils/ort_graph_to_proto.h

```diff
@@ -435,7 +438,7 @@ Ort::Status OrtGraphToProto(const OrtGraph& graph,
       } else {
         // User wants to store data inline the TensorProto's raw_data
         tensor_proto->set_data_location(onnx::TensorProto_DataLocation_DEFAULT);
-        tensor_proto->set_raw_data(data, data_bytes);
+        onnxruntime::utils::SetRawDataInTensorProto(*tensor_proto, data, data_bytes);
```

Comment:
I believe this function comes from an internal ORT file. We do not want other dependencies in this file. Please refer to this comment: https://github.com/microsoft/onnxruntime/pull/25791/files#r2325723465